src/common/duplication_common.cpp

```diff
@@ -103,6 +105,59 @@ class duplication_group_registry : public utils::singleton<duplication_group_reg
     }
 
     const std::set<uint8_t> &get_distinct_cluster_id_set() { return _distinct_cids; }
+    error_with<uint8_t> reload_duplication_config(std::string config_file)
+    {
+        if (config_file.length() == 0) {
+            config_file = "config.ini";
+        }
+        const char *config_file_cstr = config_file.c_str();
+
+        std::map<std::string, uint8_t> new_group;
+        std::set<uint8_t> new_distinct_cids;
+        dsn::configuration old_config;
+
+        // reload default config.ini, user can point to another config file. update g_config here
+        if (!dsn_config_reload(config_file_cstr, nullptr, &old_config)) {
+            LOG_ERROR("Fail to reload config file {} \n", config_file_cstr);
```

Comment:
There are some format issues, please check them.
```suggestion
            LOG_ERROR("Fail to reload config file {}", config_file_cstr);
```